internal/pkg/cli/env_init.go

```diff
@@ -69,11 +71,18 @@ func (opts *InitEnvOpts) Execute() error {
 		return err
 	}
 
+	// TODO: move to opts method?
```

Comment:
What does this todo mean?

Reply:
`opts.Prepare()` or something similar to other workflows which fills in required data to the `opts` structure.

Reply:
I see, would you mind updating the comment with this additional context?